android/engine/src/main/java/org/smartregister/fhircore/engine/task/FhirCarePlanGenerator.kt

```diff
@@ -406,7 +445,11 @@ constructor(
             )
 
           if (resourceClosureConditionsMet) {
-            defaultRepository.updateResourcesRecursively(eventResource, subject, eventWorkFlow)
+            defaultRepository.updateResourcesRecursively(
+              eventResource,
+              subject,
+              eventWorkFlow,
```

Comment:
Improve readability by using named params.

Reply:
change in code were not introduced by my commit and were there from before

Reply:
See previous comment

Reply:
OK